internal/version/version.go

```diff
@@ -1,3 +1,3 @@
 package version
 
-const Version = "1.5.0-beta3"
+const Version = "2.0.0-beta1"
```

Comment:
Our open source tools typically follow the [semantic versioning](https://semver.org/) convention. This change counts as a major version change because it may break existing implementations / usages of `ld-find-code-refs`